ci/ci/github.py

```diff
@@ -485,7 +482,7 @@ def query():
                     name: "{self.target_branch.branch.repo.name}"
                   ) {{
                     pullRequest (number: {self.number}) {{
-                      {"reviewDecision" if review_decision == -1 else ""}
+                      {"reviewDecision" if review_decision is None or review_decision == "API_NONE" else ""}
```

Comment:
Did you test the API cost of a round of updates with and without doing this? My naive guess would have been that this is a "fact about a node" rather than "an additional node" so wouldn't cost any more API coins. And it might be simpler not to have this if statement in the middle of the query string?